addons/account/models/account_invoice.py

```diff
@@ -1293,15 +1298,19 @@ class AccountInvoiceTax(models.Model):
     _order = 'sequence'
 
     def _compute_base_amount(self):
-        for tax in self:
-            base = 0.0
-            for line in tax.invoice_id.invoice_line_ids:
-                if tax.tax_id in line.invoice_line_tax_ids:
-                    price_unit = line.price_unit * (1 - (line.discount or 0.0) / 100.0)
-                    base += (line.invoice_line_tax_ids - tax.tax_id).compute_all(
-                        price_unit, line.invoice_id.currency_id, line.quantity, line.product_id, line.invoice_id.partner_id
-                    )['base']
-            tax.base = base
+        tax_grouped = {}
+        for tax in self.sorted(key=lambda r: r.sequence):
+            key = self.env['account.tax'].browse(tax.tax_id.id).get_grouping_key({
+                'tax_id': tax.tax_id.id,
+                'account_id': tax.account_id.id,
+                'account_analytic_id': tax.account_analytic_id.id,
+            })
+            if not tax_grouped:
```

Comment:
ok, but it could be more generic, just in case: https://github.com/odoo/odoo/pull/13863/commits/76ec711cbe5fb51fa141b0052a568dd024363532#diff-f3712147f4b25231caa371bd83265dbeR1302